manage/upgrading/version_specific_migration/upgrade_to_python3.rst

```diff
@@ -209,74 +217,77 @@ Precompile will be run every time you run buildout. If you want to avoid running
 ---------------------
 
 As a next step we recommend that you try to start the instance with your add-on.
-This will fail on all import errors (e.g. relative imports that are not allowed in Python 3).
-If it works you can try to install the add-on.
+This will fail on all import errors (e.g., relative imports that are not allowed in Python 3).
+If it works then you can try to install the add-on.
 
 You need to fix all issues that appear before you can do manual testing to check for big, obvious issues.
 
 
 Common Issues during startup
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The following issues will abort your startup and you need to fix them before you are able to test the functionality by hand or run tests.
+The following issues will abort your startup.
+You need to fix them before you are able to test the functionality by hand or run tests.
 
 
 A - Class Advice
 ^^^^^^^^^^^^^^^^
 
-This kind of error message
+This kind of error message:
 
 .. code-block:: shell
 
-   TypeError: Class advice impossible in Python3.  Use the @implementer class decorator instead.
+    TypeError: Class advice impossible in Python3.  Use the @implementer class decorator instead.
 
 tells you that there is a class that is using an ``implements`` statement which needs to be replaced by the ``@implementer`` decorator.
 
-Example, this kind of code:
+For example, this kind of code:
```

Comment:
I understand what the author is trying to say, but from a user point of view, using the wording "this kind of code" is confusing.
We should try to be explicit where possible or re-word it.

Reply:
This is the start of a sentence

which is finished on the next narrative line.

This is a common pattern in documentation.

Reply:
What I mean is not the "for example" I mean "kind of code"
What "kind of code"?
"kind of code" is somewhat reallly really general.
If you are not deep into that, this is maybe not really helpful.

Reply:
Reworded in b5e5ac8